### PR TITLE
Fix Product, Log Level and Extra Params

### PR DESCRIPTION
**Pull request by zsweigart**
Fix Product, Log Level and Extra Params for Android

### CODE REVIEW

File: Types.ts [hunk 1]
```diff
@@ -44,15 +44,16 @@ export enum PlaidEnvironment {
...
-    IDENTITY,
-    INCOME,
-    INVESTMENTS,
-    LIABILITIES,
-    LIABILITIES_REPORT,
-    PAYMENT_INITIATION,
-    TRANSACTIONS,
+    ASSETS="assets",
+    AUTH="auth",
+    DEPOSIT_SWITCH="deposit_switch",
```
Comment (amytang0) [inline Types.ts:49]: nice, ty

File: android/src/main/java/com/plaid/PlaidModule.kt [hunk 2]
```diff
@@ -34,20 +34,21 @@ class PlaidModule internal constructor(reactContext: ReactApplicationContext) :
...
+    private const val LOG_LEVEL = "logLevel"
+    private const val ENV = "environment"
     private const val LINK_CUSTOMIZATION_NAME = "linkCustomizationName"
     private const val TOKEN = "token"
     private const val USER_EMAIL = "userEmailAddress"
     private const val USER_NAME = "userLegalName"
     private const val USER_PHONE = "userPhoneNumber"
     private const val WEBHOOK = "webhook"
-    private const val EXTRAS = "extras"
+    private const val EXTRAS = "extraParams"
```
Comment (amytang0) [inline android/src/main/java/com/plaid/PlaidModule.kt:51]: this might need to go in another PR, we probably broke iOS

**Review by amytang0 — CHANGES_REQUESTED** [review]
Mostly LGTM, revert change for extraParams since it will break iOS

**Review by JeroenMols — APPROVED** [review]

**Comment by zsweigart** [comment]
@amytang0 I had to rename the type  in js to `extras` as well

**Review by amytang0 — APPROVED** [review]
LGTM

**Pull request merged**